src/components/ExpandableInfo.tsx

```diff
@@ -1,216 +1,155 @@
-        duration: 0.1,
+    ref
+  ) => {
+    const { isOpen, getButtonProps, getDisclosureProps } = useDisclosure({
+      defaultIsOpen: forceOpen,
+    })
+
+    const chevronFlip = {
+      collapsed: {
+        rotate: 0,
+        transition: {
+          duration: 0.1,
+        },
       },
-    },
-    expanded: {
-      rotate: 180,
-      transition: {
-        duration: 0.4,
+      expanded: {
+        rotate: 180,
+        transition: {
+          duration: 0.4,
+        },
       },
-    },
-  }
-  const showHide = {
-    collapsed: {
-      display: "none",
-    },
-    expanded: {
-      display: "inline-block",
-    },
-  }
-  const fadeInOut = {
-    collapsed: {
-      opacity: 0,
-      transition: {
-        duration: 0.1,
-      },
-    },
-    expanded: {
-      opacity: 1,
-      transition: {
-        duration: 0.4,
-      },
-    },
-  }
-  return (
-    <Card background={background} className={className}>
-      <Content>
-        {image && <GatsbyImage image={image} alt="" />}
-        <TitleContent>
-          <Question>
-            <Header>
-              <Title>{title}</Title>
-            </Header>
-            <TextPreview>{contentPreview}</TextPreview>
-          </Question>
-        </TitleContent>
-        {!forceOpen && (
-          <ButtonContainer
-            variants={chevronFlip}
-            animate={isVisible ? "expanded" : "collapsed"}
-            initial={false}
-            onClick={() => setIsVisible(!isVisible)}
-          >
-            <Icon name="chevronDown" size="36" />
-          </ButtonContainer>
-        )}
-      </Content>
-      <motion.div
-        variants={expandCollapse}
-        animate={isVisible ? "expanded" : "collapsed"}
-        initial={false}
+    }
+
+    const animateToggle = isOpen ? "expanded" : "collapsed"
+
+    return (
+      <VStack
+        border="1px solid"
+        borderColor="border"
+        borderRadius="2px"
+        p="1.5rem"
+        mb="1rem"
+        spacing="0"
+        background={background ?? "background"}
+        position="relative"
+        _hover={{
+          "& img": {
+            transform: "scale(1.08)",
+            transition: "transform 0.1s",
+          },
+        }}
+        ref={ref}
+        {...rest}
       >
-        <motion.div
-          variants={showHide}
-          animate={isVisible ? "expanded" : "collapsed"}
+        <Stack
+          justify="space-between"
+          gap={{ base: "2rem", md: "3rem" }}
+          flexDirection={{ base: "column", md: "row" }}
+          width="full"
+        >
+          {image && <GatsbyImage image={image} alt="" />}
+          <HStack gap="3rem" width="full">
+            <Box mr="1rem">
+              <HStack
+                width="full"
+                m="1rem 0"
+                sx={{
+                  img: {
+                    mr: "1.5rem",
+                  },
+                }}
+              >
+                <Heading mt="0rem" mb="0.5rem" size="md" fontWeight="600">
```

Comment:
`Heading` component used for consistency of the heading typography in the theme config.

The `size` prop is used with the `md` value to be consistent with the current theming of the site. 
- `lineHeight` value is `1.2`
- `fontSize` is `xl`

However, it should be noted that the `xl` fontSize by default from Chakra is not the same as the one from the original config provided by the site. So currently the rendering of the font size will not be correct until the theme config is fully migrated.